index.html

```diff
@@ -1420,16 +1422,20 @@ <h3>Multi-Cloud</h3>
     </section>
 
     <section id="virtual-thing">
-      <h3>Virtual Things</h3>    
-      <section class="ednote" title="TODO">
-      <h3>TODO</h3>
-      Create a description and a diagram for a <a>virtual thing</a> scenario,
-      as described in the use case for 
-      <a href="https://w3c.github.io/wot-usecases/#USE-CASES/Retail-virtual-thing">Virtual Thing</a>
+      <h3>Virtual Things</h3>   
+	<p>
+	 A virtual thing is an abstraction of a physical entity that does not have a direct physical counterpart.
+	 It can be used to describe abstract interfaces with a TD that are decoupled from individual devices. 
+	 The purpose is to decouple a functional interface that defines the behavior
+	 from the specific implementations of devices among several manufacturers.
+	 The interface remains stable and is independent from the actual implementation of things behind that interface.
+	</p> 
+	<p>
+	  The virtual thing guarantees that 
+	 devices can be replaced without affecting the implementation of the consumer, 
+	 who can continue using the device as he did before.
```

Comment:
A better wording around the gender is needed

Reply:
I don't understand your concern. Please propose text.

Reply:
We should avoid "he" since it projects a gender on the user for no good reason.  Just replace with "they", which works fine here.  I need to look at the context, but if "consumer" is the Consuming Thing, then "it" may also be appropriate.

Reply:
replacing he with a gender-neutral pronoun/subject is needed or the subject should be removed. proposal:

`who can continue using the device as before.`

Reply:
Actually, if you mean the *person* using the thing, using the word "user" would avoid confusion with Consumer.  If you mean Consumer, capitalize and use "it".